services/userService.js

```diff
@@ -0,0 +1,61 @@
+const User = require("../models/user");
+
+async function createUser(data) {
+  try {
+    const user = await User.create(data);
+    return user;
+  } catch (error) {
+    throw error;
```

Comment:
Error should be logged or handled properly before rethrowing. Simply rethrowing the error without any logging or handling can lead to loss of valuable information for debugging purposes. Consider logging the error using a logging mechanism or a centralized error handling system.